package.json

```diff
@@ -106,7 +107,7 @@
     "@types/promise-polyfill": "^6.0.3",
     "es6-promise-promise": "^1.0.0",
     "i18next": "^19.5.6",
-    "joi": "^14.3.1",
+    "joi": "^17.1.1",
```

Comment:
I think there was a change in the joi's interface that requires changes in the code. You should check if the library is ever used in this project, maybe we can remove this dependency.